evm-contracts/src/v0.6/examples/VRFBasic.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity 0.6.6;
```

Comment:
I vote to cut this contract. It doesn't really add much clarity, the D20 is already relatively simple. This just means we have twice as much example code to maintain moving forward. What do you think?

Reply:
As a learning tool, I think it's great to have. It essentially mimics our [VRF docs example](https://docs.chain.link/docs/get-a-random-number). 

If our main source of documentation was _this repo_, then I would strongly vote in favour of keeping this contract.

However, we have external docs for that, so I'm happy to remove this.